src/Symfony/Component/HttpKernel/Controller/ArgumentResolver.php

```diff
@@ -40,12 +40,11 @@ public function __construct(ArgumentMetadataFactoryInterface $argumentMetadataFa
         $this->argumentValueResolvers = $argumentValueResolvers ?: self::getDefaultArgumentValueResolvers();
     }
 
-    public function getArguments(Request $request, callable $controller): array
+    public function getArguments(Request $request, callable $controller, \ReflectionFunctionAbstract $reflector = null): array
```

Comment:
Shouldn't the new argument be added in the interface (in phpdoc, not in the actual signature, for BC reasons) ?

Reply:
works for me, updated